imgix/urlhelper.py

```diff
@@ -14,6 +14,43 @@
 
 
 class UrlHelper(object):
+    """
+    Helper class to create single domain imgix URLs. Example:
+
+      >>> str(UrlHelper('demos.imgix.net', '/bridge.png', opts={'w': 100}))
+      https://demos.imgix.net/bridge.png?w=100
+
+    Parameters
+    ----------
+    domain : str
+    path : str
+    scheme : { 'https', 'http' }
+    sign_key : str or None
+        When provided, this key will be used to sign the generated image URLs.
+        You can read more about URL signing on our docs:
+        https://docs.imgix.com/setup/securing-images
+    sign_mode : `SIGNATURE_MODE_QUERY`
+        If `SIGNATURE_MODE_QUERY`, sign the whole URL.
+        (default `SIGNATURE_MODE_QUERY`)
+    sign_with_library_version : bool
```

Comment:
Another note for the future: we should try to make this option name consistent across all of our libraries. I'd lean towards `include_library_param` or `include_library_version`, since we're not really signing anything here.